src/NavBar/NavBar.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './NavBar.css';
+
+class NavBar extends Component {
+	render () {
+		return (
+			<div className="nav-bar">
+				<div className="nav-bar__item">Submit a book</div>
```

Comment:
Consider making these anchor tags because eventually, they will link to other pages.
`<a className="nav-bar__item" href="/submit">Submit a book<a>`